Sources/WalletConnectAuth/Auth/Auth.swift

```diff
@@ -0,0 +1,251 @@
+
+import Foundation
+import WalletConnectUtils
+import WalletConnectRelay
+import Combine
+
+public class Auth {
+    static public let instance = Auth()
+    private let client: AuthClient
+    private static var config: Config?
+    public let logger: ConsoleLogging
```

Comment:
Why logger is public here?

Reply:
I thought that during sdk integration it may be sometimes convenient to set logging level to debug.
make sense?